docs/reference/contributing/guidelines/workflow.md

```diff
@@ -90,6 +90,8 @@ Release: feature
 
 Updating target implementation (adding a new target or updating already supported target) is a change for a patch release.
 
+The test report must be part of the pull request for a new target addition. It must pass all Mbed OS functional and system validation tests for the current Mbed OS major release including all Mbed OS supported toolchains.
```

Comment:
nitpick : "The test report" implies you have referred to it before, which doesn't seem to be the case. Maybe "a test report for the new target" instead. 
and also
 It must pass all Mbed OS functional and system validation ==> The new target must pass all Mbed OS functional and system validation

Reply:
Will fix, cant find test report anywhere. We might encourage testing but t hat will be another PR if any